modules/ROOT/pages/munit-scaffold-test-task.adoc

```diff
@@ -3,10 +3,16 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
```

Comment:
Perhaps the audience confusion is coming from the title: Scaffold a Test?
Does that imply that this task is actually creating that test, rather than creating a template for writing the test?
Maybe the term "scaffold" isn't as apparent to our audience as we think?